src/Entities/Collection.ts

```diff
@@ -79,4 +79,10 @@ export class Collection {
   })
   @Column({ default: false })
   show_on_editorial: boolean = false
+
+  @Field(type => Boolean, {
+    description: "Collection has prioritized connection to artist",
```

Comment:
I'd say let's be a bit more explicit in the description `Collection should be featured on artist page`

Reply:
I intentionally made this more vague, so that we indicate that it is a high priority in relationship to its affiliated artist, but potentially leave it open to other contexts where this flag could be relevant.

Reply:
For example, we could display a similar rail on an artwork page to indicate other works by an artist, in which case the connection to the artist remains true, but the context of the artist page does not.

Reply:
Got it, sounds good then